test/primer/classify_test.rb

```diff
@@ -167,11 +167,15 @@ def test_col
   end
 
   def test_border
-    assert_generated_class("border-left",    { border: :left })
-    assert_generated_class("border-top",     { border: :top })
-    assert_generated_class("border-bottom",  { border: :bottom })
-    assert_generated_class("border-right",   { border: :right })
-    assert_generated_class("border-y",       { border: :y })
+    assert_generated_class("border-left",     { border: :left })
+    assert_generated_class("border-top",      { border: :top })
+    assert_generated_class("border-bottom",   { border: :bottom })
+    assert_generated_class("border-right",    { border: :right })
+    assert_generated_class("border-y",        { border: :y })
+    assert_generated_class("border-top-0",    { border: :top_0 })
```

Comment:
I _think_ this should be `border_top: 0`, to match Styled System: https://styled-system.com/table/#border. I'd usually ask @emplums, but she is OOO until Monday 😄

Reply:
@joelhawksley Just pushed up new changes to change it to border_top: 0